test-integration/helpers/reading-builder.coffee

```diff
@@ -269,10 +273,11 @@ class ReadingBuilder extends TestHelper
           @cancel() if isDisplayed
       else
         # Click "Add Readings"
+        @test.sleep(100) # Not sure why this is needed
         @el.addReadingsButton.click()
```

Comment:
Oh, so I could maybe call `wait.click()` and have the selector use `:not(.is-waiting)` (or something)?